features/environment.py

```diff
@@ -273,7 +240,7 @@ def __init__(
         timed_job_tag += str(job_suffix)
         timed_job_tag = timed_job_tag.replace(".", "-")
 
-        # Add 8-digit random suffix to the tag, so it does not conflict when
+        # Add 7-digit random suffix to the tag, so it does not conflict when
```

Comment:
Isn't this 8? `k=8` below should make it 8

Reply:
Yeah, this is was definitely a mistake. I should have not changed that number. Thanks for catching that